CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ### vNEXT
 
+### 0.5.4
```

Comment:
These should just be placed under vNEXT as multiple different merge requests may get bundled into the next release.